packages/core/src/sdk/graphql/request.ts

```diff
@@ -14,73 +15,90 @@ export interface BaseRequestOptions<V = any> {
   fetchOptions?: RequestInit
 }
 
-const DEFAULT_HEADERS_BY_VERB: Record<string, Record<string, string>> = {
-  POST: {
-    'Content-Type': 'application/json',
-  },
-}
-
-export const request = async <Query = unknown, Variables = unknown>(
-  operation: Operation,
-  variables: Variables,
-  options?: RequestOptions
-) => {
-  const { data, errors } = await baseRequest<Variables, Query>('/api/graphql', {
-    ...options,
-    variables,
-    operation,
-  })
-
-  if (errors?.length) {
-    throw errors[0]
-  }
-
-  return data
+const MethodByOperation = async (operationName: string) => {
+  const { experimental } = await import('../../../discovery.config')
+  return Array.isArray(experimental.cachedOperations) &&
+    experimental.cachedOperations.includes(operationName)
+    ? 'GET'
+    : 'POST'
 }
 
-/* This piece of code was taken out of @faststore/graphql-utils */
-const baseRequest = async <V = any, D = any>(
+/* This piece of code was taken out of @vtex/faststore-graphql-utils */
```

Comment:
```suggestion
/* This piece of code was taken out of @faststore/graphql-utils */
```

I think this was before undoing the renaming 🤔